package.json

```diff
@@ -14,6 +14,8 @@
   ],
   "scripts": {
     "build": "yarn rimraf dist && tsc",
+    "summarize": "ts-node scripts/generateSummary.ts",
+    "summarize-npx": "npx ts-node scripts/generateSummary.ts",
```

Comment:
This should not be required as ts-node is a dev dependency

Reply:
Actually, it's interesting that it's a dev dependency. It's not used anywhere

Reply:
Removed: https://github.com/safe-global/safe-deployments/pull/228/commits/1eecd7bda874c4f7a1e1ad41ecfec37ad1dbd210

Reply:
Why?

Reply:
I thought Richard wanted us to remove it.

> This should not be required as ts-node is a dev dependency

Reply:
I mentioned in the comment that it's not used anywhere, so it's an open question whether we want to keep it as a dependency. Since it is not related to the development of the package and is only used in the helper script to generate a table, I personally think it shouldn't be a dev dependency of the package.